src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -321,6 +321,42 @@ def _uploadJobDependencies(self, arcJobID, executableFile, inputs, executables):
             self.log.verbose("Input correctly uploaded", fileToSubmit)
         return S_OK()
 
+    def _killJob(self, arcJobList):
+        """Kill the specified jobs
+
+        :param list arcJobList: list of ARC Job IDs
+        """
+        if not self.session:
+            return S_ERROR("REST interface not initialised. Cannot kill jobs.")
+
+        # Get a proxy
+        result = self._prepareProxy()
+        if not result["OK"]:
+            self.log.error("Failed to set up proxy", result["Message"])
+            return result
+        self.session.cert = Locations.getProxyLocation()
+
+        # List of jobs in json format for the REST query
+        jobsJson = {"job": [{"id": job} for job in jobIDList]}
```

Comment:
Oops, it should be changed here too. This should fix the problems with the CI: 
```suggestion
        jobsJson = {"job": [{"id": job} for job in arcJobList]}
```